src/client-configuration.ts

```diff
@@ -120,6 +120,11 @@ export interface ClientConfiguration {
    */
   typecheck?: boolean;
 
+  /**
+   * Enable or disable performance hints. If no value is provided, performance hints will be disabled.
```

Comment:
Can we add the context that this shows up in QueryInfo's summary? Otherwise, it's not very clear how this changes the output.